modules/local/clipkit.nf

```diff
@@ -0,0 +1,37 @@
+process CLIPKIT {
+    tag "${meta.og}"
+    label 'process_medium'
+
+    container "${ workflow.containerEngine == 'docker' ? 'docker.io/austinhpatton123/clipkit' :
```

Comment:
at some point we should discuss making the actual Dockerfiles publicly available in our Arcadia Org github. It adds a layer of transparency so people know what is actually in the Dockerfile and then makes it easier to make updates over time

Reply:
Yes, we definitely should!